docs/cmd/kn_plugin.md

```diff
@@ -4,10 +4,9 @@ Plugin command group
 
 ### Synopsis
 
-Provides utilities for interacting and managing with `kn` plugins.
-
-Plugins provide extended functionality that is not part of the core `kn` command-line distribution.
+Provides utilities for interacting and managing with kn plugins.
 
+Plugins provide extended functionality that is not part of the core kn command-line distribution.
```

Comment:
this came part of `./hack/build.sh`

Reply:
Hmm, I don't understand why this file changed with your PR

Reply:
Looks like restructure docs PR should've these changes, running hack/buildsh.sh on current master returns this diff

```
12:45 ➜  client git:(master)  ./hack/build.sh 
🕸️ Update
🧽 Format
⚖️ License
📖 Docs
🚧 Compile
🧪 Test
────────────────────────────────────────────
Version:      v20191001-local-ffbf684-dirty
Build Date:   2019-10-01 07:15:07
Git Revision: ffbf684
Support:
- Serving: v0.8.0  v0.7.1
- API(s):  v1alpha1
12:45 ➜  client git:(master) ✗  git --no-pager diff                   
diff --git a/docs/cmd/kn_plugin.md b/docs/cmd/kn_plugin.md
index 8d6311a..5d17d26 100644
--- a/docs/cmd/kn_plugin.md
+++ b/docs/cmd/kn_plugin.md
@@ -4,10 +4,9 @@ Plugin command group
 
 ### Synopsis
 
-Provides utilities for interacting and managing with `kn` plugins.
-
-Plugins provide extended functionality that is not part of the core `kn` command-line distribution.
+Provides utilities for interacting and managing with kn plugins.
 
+Plugins provide extended functionality that is not part of the core kn command-line distribution.
 Please refer to the documentation and examples for more information about how write your own plugins.
 
 ```
@@ -34,3 +33,4 @@ kn plugin [flags]
 
 * [kn](kn.md)	 - Knative client
 * [kn plugin list](kn_plugin_list.md)	 - List plugins
+
12:45 ➜  client git:(master) ✗  

``